Sources/ProcessOut/Sources/Repositories/Shared/Models/PONativeAlternativePaymentMethodParameter.swift

```diff
@@ -22,6 +22,19 @@ public struct PONativeAlternativePaymentMethodParameter: Decodable {
 
         /// For phone fields.
         case phone
+
+        /// For fields where user needs to select one of available values.
+        case singleChoice = "single-choice"
```

Comment:
I think it should be `single-select`
https://checkout.atlassian.net/browse/POINT-923